test/wpt/webml/idlharness.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Web ML Polyfill API IDL Tests</title>
+<script src="../resources/testharness.js"></script>
```

Comment:
What's the license of files inside `resources`? If they are from third_party, we'd better to put a README.md to explain.